src/content_handlers/apt_handler/src/apt_parser.cpp

```diff
@@ -67,6 +67,10 @@ std::unique_ptr<AptContent> GetAptContentFromRootValue(JSON_Value* rootValue)
     aptContent->Name = aptName;
     aptContent->Version = aptVersion;
 
+    // Note: json_object_get_boolen returns -1 on fail.
```

Comment:
>json_object_get_boolen [](start = 13, length = 22)

typo

Reply:
fix

---
In reply to: [596182740](https://github.com/Azure/iot-hub-device-update/pull/17#discussion_r596182740) [](ancestors = 596182740)